### DISCUSSION TITLE
Problem z zapisaniem ssid/ password

### DISCUSSION THREAD

**Discussion by lauba1** [hackerspace-silesia/Smogomierz#154] [General]
Ma ktos problem identyczny jak w tym watku: https://github.com/alanswx/ESPAsyncWiFiManager/issues/102
U mnie po flash erase, ssid i password nie jest zapisywany w NVM.

**Comment by SajkMajk** [comment]
Mam podobnie. W wersji 2.8.1 na ESP8266 co 24 godziny tracę połączenie z routerem i mój Smogly przechodzi do trybu konfiguracji sieci wifi.

**Reply by hanskiepl** [reply]
potwierdzam. Wszystko było ok chyba do momentu kiedy chciałem ustawić sobie wysłąnie danych do serwera MQTT.... i się posypało.

**Reply by lukaszog** [reply]
@SajkMajk udało Ci się rozwiązać problem?

**Reply by SajkMajk** [reply]
> @SajkMajk udało Ci się rozwiązać problem?

Jeszcze nie. Potrzebowałem sprzętu pod inny projekt, ale ponieważ temat smogu w mojej dzielnicy powraca, postawię czujnik od nowa i dam znać jakie efekty.

**Reply by hanskiepl** [reply]
Pod tym postem umieściłem korektę problemu z wifi.
Reszty nie ruszałem - pozostałych problemów nie ogarnę.
https://github.com/hackerspace-silesia/Smogomierz/issues/151#issuecomment-1312807538

**Comment by LeanderLBN** [comment]
Czy jest jakieś rozwiązanie tego problemu? Zadowolony z siebie zmontowałem i powiesiłem smogomierz i... no niestety. Codzienna utrata połączenia z wifi.
